### PR TITLE
Fix false positives on import statements

### PR DESCRIPTION
**Pull request by heliocodacy**

### CODE REVIEW

**Review by DMarinhoCodacy — APPROVED** [review]
LGTM!

**Comment by lolgab** [comment]
Can you add a integration test which fails without the new addition?

**Comment by heliocodacy** [comment]
> Can you add a integration test which fails without the new addition?

✅

File: docs/duplication-tests/ts-ignore-imports/src/file1.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    '123o',
+    '123p',
+    '123q',
+    '123r',
+    '123s',
+    '123t',
+    '123v',
+    '123z',
+  ]
+};
```
Comment (lolgab) [inline docs/duplication-tests/ts-ignore-imports/src/file1.ts:32]: Is this part necessary?
Let's keep the test files minimal

File: docs/duplication-tests/ts-ignore-imports/src/file2.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    '123b',
+    '123c',
+    '123d',
+    '123e'
+  ]
+};
+
+function aPossibleFunction(n: number): number {
+  return n + 1
+}
```
Comment (lolgab) [inline docs/duplication-tests/ts-ignore-imports/src/file2.ts:43]: Is this necessary?

**Comment by lolgab** [comment]
Great fix! This import thing has been annoying for a long time!
Kudos!

**Pull request merged**